eos-updater-flatpak-installer/eos-updater-flatpak-installer.service.in

```diff
@@ -34,15 +35,17 @@ PrivateUsers=yes
 ProtectControlGroups=yes
 ProtectHome=yes
 ProtectKernelModules=yes
-ProtectKernelTunables=yes
+# bwrap also mounts /proc
+ProtectKernelTunables=no
```

Comment:
Could we improve the sandboxing in bubblewrap so it doesn’t mount `/proc` writeable? It seems like there should be no need for the flatpak triggers to have a writeable `/proc`.

Reply:
Hmm... To be honest I'm not 100% sure why this ended up being needed. There was a failure bind mounting /proc into the new root. I assumed that ProtectKernelTunables somehow prevented /proc being mounted, but it doesn't seem to - it's just adding read-only bind mounts on top of /proc and /sys. So maybe the problem is not the read/write-ness, but to do with recursive bind mounting?

Reply:
[`ProtectKernelTunables=`](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectKernelTunables=) is essentially `ReadOnlyPaths=/proc`, which has a [paragraph](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ReadWritePaths=) in its documentation about propagation of mounts up to the main mount namespace, which would be relevant if it weren’ŧ for the fact that bwrap uses a mount namespace (right?). So I’m stumped.